src/main/java/net/minecraftforge/event/entity/EntityTeleportEvent.java

```diff
@@ -64,7 +65,7 @@ public EntityTeleportEvent(Entity entity, double targetX, double targetY, double
 
     /**
      * EntityTeleportEvent.TeleportCommand is fired before a living entity is teleported
-     * from use of {@link net.minecraft.command.impl.TeleportCommand}.
+     * from use of {@link net.minecraft.server.commands.TeleportCommand}.
```

Comment:
Import inconsistency.

Reply:
This cannot be changed to an import, as it would then conflict with the `TeleportCommand` inner class.